### PR TITLE
minor rdb changes

### PR DESCRIPTION
**Pull request by dsx-**

### CODE REVIEW

**Comment by AmbientMalice** [comment]
There's this weird tradition/convention PJ64's RDB has where everything is in lower case. I tried to stick to it except I slipped up a few times. It's weird. If it's a plugin note, all lower case. If it's a Core Note, the first word will usually be capitalised.

**Comment by dsx-** [comment]
yeah these changes were for consistency

**Comment by AmbientMalice** [comment]
Any reason you've closed this?

**Comment by dsx-** [comment]
ive got more changes to make and i was going to include these changes with them in a new pull request... or should i have two pull requests

**Comment by AmbientMalice** [comment]
I ask because I'm working on a PR of my own and didn't want to tread on your toes.

**Comment by dsx-** [comment]
pushing some minor fixes in a sec...for core notes theres currently a mix of uppercase first letters and lowercase, which should we do?

**Comment by AmbientMalice** [comment]
I personally think it looks cleaner if it goes like...

Core Note: Big problem

Plugin Note [video/Glide64] issue at hand

Keeping with PJ64's traditional RDB aesthetic, I think that lowercase following brackets looks good, but if there's no brackets preceding, the first word should be capitalised. Also, acronyms should be capitalised wherever appropriate.

**Comment by dsx-** [comment]
ok cool

**Comment by dsx-** [comment]
alright im done

**Comment by AmbientMalice** [comment]
Good job.

File: Config/Project64.rdb [hunk 1]
```diff
@@ -8133,4 +8113,4 @@ Good Name=MGC 2011 Demo (PD)
 Internal Name=shut up and code
 CPU Type=Interpreter
 RDRAM Size=8
-Status=Unsupported
+Status=Unsupported
```
Comment (cxd4) [inline Config/Project64.rdb]: noooooooooooooooo!

That poor end-of-file line break :sob: , slaughtered like the helpless sacrificial semantic entity that he is.

Comment (vgturtle127) [inline Config/Project64.rdb]: @dsx- Other than adding another line break at the end, I see nothing wrong with merging this.

Maybe change this:

Rat Attack
Core Note=Interpreter only for multiplayer

Other than that, it looks great. :smile:

**Comment by dsx-** [comment]
change what

yeah deleted the line break without thinking i guess...why does there have to be a line break anyway, seems unnecessary

**Comment by cxd4** [comment]
It's more of a traditional thing just because, every line ends with a "end-of-line" character, of sorts.

It also is easier to define on text parsing in C programs.  If a C program scans a file stream for a full line of text, it reads up until the end of a line (meaning, the line break in question) or until something pseudo-erroneous happens, like the end of the file being reached.  Therefore it's "cleaner" to always have even the final line of a text document end with a line break.

I'm not that bothered by it, but GitHub does seem to troll about it as well:
https://github.com/project64/project64/pull/918/files#diff-6ed817341fe13121d9cae89c071156ccL8136
... where that red symbol gets added at the end, complaining "no newline at end of file"

**Comment by cxd4** [comment]
http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

The long and short of it is that the human eye assumes when a line ends by looking at the absence of continuing text on that line, but in terms of algorithm, every line is delimited by a line break marking the end of each line--even the final line at the bottom of the text file.  Otherwise, you risk parser errors running off the end of the file's total size because it never found a line break to cut the line parsing off.

**Comment by vgturtle127** [comment]
@dsx- I meant change the "core note" line to what I specified, which was "Core Note=Interpreter only for multi" before and "Core Note=Interpreter only for multiplayer" after I changed it. It just makes it more clear.

**Comment by dsx-** [comment]
OHHH right yep sorry

**Comment by vgturtle127** [comment]
@dsx- It's all good, man. :smile: 

I can see people getting confused, since "multi" can mean many things when talking about video games or computer graphics.

**Pull request merged**